components/explore/ExploreSort.vue

```diff
@@ -33,16 +32,15 @@
         v-for="option in options"
         :key="option"
         aria-role="listitem"
+        class="is-flex"
         :value="option">
         <span>
           {{
             $i18n.t(isItems ? `sort.${option}` : `sort.collection.${option}`)
           }}
         </span>
-        <img
-          v-if="selectedSort.includes(option)"
-          class="sort-check"
-          src="/checkmark.svg" />
+        <o-icon v-if="selectedSort.includes(option)" icon="check" class="ml-2">
```

Comment:
@preschian dont we have some neoicon component?

Reply:
@vikiival not yet, I added it in #5237

Reply:
https://github.com/kodadot/nft-gallery/pull/5237 is merged Please proceed